mir-module/src/main/resources/xsl/metadata/mir-abstract.xsl

```diff
@@ -88,6 +88,17 @@
             <xsl:with-param name="query" select="concat('&amp;fq=link:*',$accessCondition, '&amp;owner=createdby:', $owner)" />
           </xsl:call-template>
         </xsl:if>
+          <xsl:if test="not(mcrxsl:isCurrentUserGuestUser())">
```

Comment:
but isn't this stylesheet used for the metadata page and not for the result list?